lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -428,27 +428,29 @@ private static function get_responsive_container_markup( $attributes, $inner_blo
 		$responsive_dialog_directives    = '';
 		$close_button_directives         = '';
 		if ( $should_load_view_script ) {
-			$open_button_directives          = '
```

Comment:
I checked and these changes are still included in the `WP_Navigation_Block_Renderer` which has now been moved to the block-library package and thus will be automatically synced to Core as part of the packages sync process.